sci-biology/cufflinks/files/cufflinks-2.2.1-gcc6.patch

```diff
@@ -0,0 +1,14 @@
+--- a/src/lemon/error.h
++++ b/src/lemon/error.h
```

Comment:
> cufflinks is awful C++, and either way both ways are the wrong way to handle errors

Agreed.

> Maybe use the debian patch instead

Debian's patch returns `NULL` in a copy assignment operator returning a reference to a class type.  The only reason it compiles is because it is in a class template and there is no use of, and thus no instantiation of, the assignment operator in the code.  The behavior is just as undefined as the original.

It doesn't really matter though, as both approaches would be insufficient for exception handling and the method is never actually called.